.eslintrc.yml

```diff
@@ -4,6 +4,7 @@ parserOptions:
 env:
   es6: true
   node: true
+  es2020: true
```

Comment:
Otherwise looks great to me

Reply:
we can improve the eslint config in a follow-up PR.

also cc @B2o5T as the eslint expert 🚀

Reply:
@yaacovCR is right, you can remove `es6`, you can even remove `parserOptions.ecmaVersion: 2020` because when you setup `env.es2020: true` ESLint set automatically `ecmaVersion` parser option to 11 (alias for 2020).

<img width="774" alt="image" src="https://user-images.githubusercontent.com/7361780/162328904-55128321-7913-42f1-8f57-941996651ef8.png">


Also, I think you can use even the latest available ECMAScript version - `env: 2022`, because anyway the source code will be transpiled by babel.

And the latest to make graphql cross-platform friendly you should remove top-level `env.node: true` and apply this option only on node-only files via the `overrides` section with something like this:

```yml
overrides:
  - files:
      - website/docusaurus.config.js
      - website/sidebars.js
      - integrationTests/**/*
      - benchmark/**/*
      - resources/**/*
    env:
      node: true
```